### ISSUE TITLE
**IDEA** resample only 'long' events for improved efficiency

### ISSUE THREAD

**Issue by remi-daigle** [rystanley/TurtleSimulations#1]
OK, so it looks as though sampling efforts shorter than 120 mins are pretty useless, what if we do the resampling exercise using only the events longer than 120mins?

**Comment by rystanley** [comment]
That is true, however I think that is captured in the boxplot return. In reality they wanted to know what the return on investment is, so to speak. So the cumulative plots will account for the relatively fruitless sampling events which are shorter than 120 minutes. I think the limitation of the analysis so far is that we make the assumption that all turtles are available in all years. However it would appear that it took them approximately 900 observational hours before the number of new turtles reaches a asymptote and the additional turtles are likely new recruits.

**Comment by remi-daigle** [comment]
yeah, I'm thinking that if we perform a resampling on the >120 min sampling events, we may be able to sample all the turtles in less than 900 hours. I'm am basing this off those boxplots since it looks to me like sampling events >120 mins have very poor return on investment

**Comment by rystanley** [comment]
Sure in that case I think we would be getting at the idea of how efficient could they be. However the plot as it stands now is how has the collective monitoring to date contributed towards the assessment of the population (Figs 6 & 7) which we crudely put somewhere between 30-52 individuals with some recruitment shifting that number. The issue we have to deal with before we use re-sampling to get at the population estimate is what is the baseline that monitoring is trying to reach. We need to modify this code so that new individuals an come in (move the population up). They have crazy low mortality which is effectively 0 within the span that we have sampled. The other issue for this to work, which I view as another contract, would be to see how much sampling effort do they need to do to capture a turtle at the frequency needed to say it's 'alive' (e.g. Once they see it once is that sufficient or do they need to see it every year, 2 years, ... ). Something to think about. Maybe take the distribution of turtles and how many times they were observed on average over a sliding window of time (5 years) then come up with an average return on sampling, and randomly re-sample turtles from that weighted distribution. 

Things to do for sure but we would need more feed back from them (e.g. what is a plausible regular sampling interval).
